state-chain/traits/src/mocks/deposit_handler.rs

```diff
@@ -65,7 +65,7 @@ impl<C: Chain, T: Chainflip> MockDepositHandler<C, T> {
 				),
 				ForeignChain::Bitcoin => todo!("Bitcoin address"),
 			},
-			1u128,
+			Default::default(),
```

Comment:
This is change in behaviour - If we expected this to be one 1 anywhere, we could use One::one .